packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.pcss

```diff
@@ -1,208 +1,608 @@
 atomic-result-icon {
-    display: block;
-    width: var(--atomic-icon-size);
-    height: var(--atomic-icon-size);
-    background-size: var(--atomic-icon-size) var(--atomic-icon-size);
-    border-radius: 2px;
+  display: block;
+  width: 100%;
+  height: 100%;
+  background-size: 100% 100%;
```

Comment:
Alternatively, the icon could take a size like `2rem` (32px on a font-size of 16px) by default. 

The only particularly important line is the one that sets background-size to 100%.